### ISSUE TITLE
error installing

### ISSUE THREAD

**Issue by rauhryan** [fivetanley/ember-cli-migrator#50]
```
ryanrauh ~/lecode/huboard-rails/ember-migration $ npm install -g ember-cli-migrator
npm WARN engine makeerror@1.0.10: wanted: {"node":"0.6.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine tmpl@1.0.3: wanted: {"node":"0.6.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
/Users/ryanrauh/.nodenv/versions/0.12.0/bin/ember-cli-migrator -> /Users/ryanrauh/.nodenv/versions/0.12.0/lib/node_modules/ember-cli-migrator/bin/ember-cli
-migrator
npm WARN unmet dependency /Users/ryanrauh/.nodenv/versions/0.12.0/lib/node_modules/ember-cli-migrator/node_modules/ember-cli/node_modules/broccoli-es6modul
es/node_modules/esperanto requires chalk@'^1.0.0' but will load
npm WARN unmet dependency /Users/ryanrauh/.nodenv/versions/0.12.0/lib/node_modules/ember-cli-migrator/node_modules/ember-cli/node_modules/chalk,
npm WARN unmet dependency which is version 0.5.1
ember-cli-migrator@0.6.0 /Users/ryanrauh/.nodenv/versions/0.12.0/lib/node_modules/ember-cli-migrator
├── walk-sync@0.1.3
├── underscore.string@2.4.0
├── commander@2.7.1 (graceful-readlink@1.0.1)
├── rsvp@3.0.17
├── chalk@1.0.0 (escape-string-regexp@1.0.3, ansi-styles@2.0.1, supports-color@1.3.1, strip-ansi@2.0.1, has-ansi@1.0.3)
├── mkdirp@0.5.0 (minimist@0.0.8)
├── recast@0.8.8 (cls@0.1.5, private@0.1.6, depd@1.0.0, ast-types@0.5.7, esprima-fb@7001.1.0-dev-harmony-fb, source-map@0.1.32)
└── ember-cli@0.2.0 (abbrev@1.0.5, ember-cli-copy-dereference@1.0.0, promise-map-series@0.2.1, js-string-escape@1.0.0, rimraf@2.2.8, temp@0.8.1, marked@0.$.3, exit@0.1.2, http-proxy@1.9.0, debug@2.1.3, symlink-or-copy@1.0.1, yam@0.0.18, broccoli-writer@0.1.1, broccoli-sane-watcher@0.1.1, marked-terminal@1.2.$, broccoli-sourcemap-concat@0.4.3, broccoli-caching-writer@0.5.1, broccoli-es6modules@0.5.1)
```

node: 0.12.0 
npm: 2.5.1

```
ryanrauh ~/lecode/huboard-rails/ember-migration $ ember-cli-migrator -g App -t dest -s source -d huboard
require("recast").Visitor deprecated Please use require("recast").visit instead of require("recast").Visitor ../../../.nodenv/versions/0.12.0/lib/node_modu
les/ember-cli-migrator/lib/helper-visitor.js:26:28
require("recast").Visitor deprecated Please use require("recast").visit instead of require("recast").Visitor ../../../.nodenv/versions/0.12.0/lib/node_modu
les/ember-cli-migrator/node_modules/recast/node_modules/cls/main.js:91:71
require("recast").Visitor deprecated Please use require("recast").visit instead of require("recast").Visitor ../../../.nodenv/versions/0.12.0/lib/node_modu
les/ember-cli-migrator/lib/migrator-visitor.js:8:30
module.js:338
    throw err;
          ^
Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/ryanrauh/.nodenv/versions/0.12.0/lib/node_modules/ember-cli-migrator/lib/ember-migrator.js:15:14)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

**Comment by rauhryan** [comment]
I was able to install @0.5.0

**Comment by rauhryan** [comment]
same deprication warnings but the command actually runs

**Comment by fivetanley** [comment]
ah scumbag me for forgetting to add lodash :P

**Comment by rauhryan** [comment]
oh no worries

![https://s3.amazonaws.com/f.cl.ly/items/2j2A45322J2b2z1q3346/Image%202015-03-16%20at%204.00.20%20PM.png](https://s3.amazonaws.com/f.cl.ly/items/2j2A45322J2b2z1q3346/Image%202015-03-16%20at%204.00.20%20PM.png)

trying it for the ember-tools port, it did a pretty good job moving files and its almost there on converting them

would it be worth it to add support for transforming the browserify old school require syntax into es6?

maybe not if I'm like the only person alive using ember-tools

**Comment by fivetanley** [comment]
I think it'd be a pretty easy change to make, I can walk you through how I'd attempt to do it this week if you want another pair of eyes. Would be totally open to a PR though.

**Comment by fivetanley** [comment]
I released 0.6.1 to get rid of that warning (I hope). I opened https://github.com/fivetanley/ember-cli-migrator/issues/51 for ember-tools so it's easier for me to track.

**Comment by rauhryan** [comment]
somewhere in the here code here maybe????

https://github.com/fivetanley/ember-cli-migrator/blob/master/lib/migrator-visitor.js#L31-L52
